libs/community/langchain_community/vectorstores/chroma.py

```diff
@@ -357,13 +370,15 @@ def similarity_search_by_vector(
         k: int = DEFAULT_K,
         filter: Optional[Dict[str, str]] = None,
         where_document: Optional[Dict[str, str]] = None,
+        include_id: Optional[bool] = False,
```

Comment:
We'll need to be careful in terms of how we deal with the ID, so it can be rolled out throughout the various integrations.